nilearn/_utils/cache_mixin.py

```diff
@@ -108,11 +108,11 @@ def cache(func, memory, func_memory_level=None, memory_level=None,
     memory: instance of joblib.Memory or string
         Used to cache the function call.
 
-    func_memory_level: int, optional
+    func_memory_level: int
```

Comment:
Is there a standard way to precise that in the doc?

Also, I'm not fond of this name but it already changed twice...